scripts/console.ts

```diff
@@ -410,22 +426,80 @@ function etherscanTx(net: string, txHash: string): string {
 }
 
 function askForUsage(): string {
-    const usage = [CLONE_USAGE, DEV_USAGE];
-    const network = keyInSelect(usage, `Please enter your intended usage`, { cancel: true });
-    return usage[network];
+    const usageOpts = [CONTRIBUTOR_USAGE, MAINTAINER_USAGE, DEV_USAGE];
+    const usageChoice = keyInSelect(usageOpts, `Please enter your intended usage`, {
+        cancel: true,
+    });
+
+    switch (usageOpts[usageChoice]) {
+        case CONTRIBUTOR_USAGE:
+            console.log(
+                `\n\nYou will be asked to enter the address of the already deployed GitConsensus, Token, or Governor ` +
+                    `contracts that you want to use on ${network.name}. These will be defaulted from the list of the official deployed ` +
+                    `contracts on ${network.name} which can be found in this repository's 'deployments.json' file.`,
+            );
+            break;
+        case MAINTAINER_USAGE:
+            console.log(
+                `\nTo turn your new or existing git project into a git project utilizing the Git Consensus Protocol, ` +
+                    `you will need to go through a one-time step of deploying project-specific Token & Governor clones.` +
+                    `\n\nThe Token clone is an extended ERC20 contract that holds balances for each address, and ` +
+                    `assigns voting power based on this balance.` +
+                    `\n\nThe Governor clone is a standard Governor contract that utilizes the Token's voting balance to create, ` +
+                    `vote on, and execute proposals.` +
+                    `\n\nBoth of these clones accept input parameters that are specific to your project. Defaults have been ` +
+                    `provided, which you can accept by pressing enter.` +
+                    `\n\nYou will also be asked to enter the address of the already deployed GitConsensus, TokenFactory, and GovernorFactory ` +
+                    `contracts that you want to use on ${network.name}. These will be defaulted from the list of the official deployed ` +
+                    `contracts on ${network.name} which can be found in this repository's 'deployments.json' file.`,
+            );
+            break;
+        case DEV_USAGE:
+            console.log(
+                `\nAlthough the of the core Git Consensus contracts (GitConsensus, TokenFactory, and GovernorFactory) ` +
+                    `have been deployed to most networks, contract developers may also wish to deploy their own versions of ` +
+                    `these contracts. As long as these contracts implement the interfaces (e.g. IGitConsensus), various other ` +
+                    `logic can be adjusted.` +
+                    `\n\nKeep in mind, any newly deployed contracts will have clean state (e.g. an empty 'commitToOwnerAddr' ` +
```

Comment:
Since we're thinking of combining the commit to owner map with the tag to token map, it might be better to use a different example for state, so that we don't forget to rename `commitToOwnerAddr` to `hashesToOwner`, for example.

Reply:
good call!